_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -0,0 +1,143 @@
+
+Before searchable snapshots, searching the data in remote snapshots required restoring the snapshots to the OpenSearch cluster node. Thus, in order to search snapshots, you had to expand your infrastructure with larger OpenSearch nodes. 
+
+The searchable snapshots feature caches frequently used data segments in cluster nodes, removing the least used data segments from the cluster nodes to make space for frequently used ones. The data segments downloaded from snapshots on block storage reside alongside the general indexes of the cluster nodes. As a result, the computing capacity of cluster nodes is shared between indexing, local search, and data segment search in a snapshot that resides in lower-cost object storage, like Amazon Simple Storage Service (Amazon S3). While cluster node resources are utilized much more efficiently, the large number of tasks results in slower snapshot searches. Additionally, the local storage of the node is used for caching the snapshot data. Therefore, you can search snapshots stored in lower-cost storage, like Amazon S3, with acceptable performance.
+
+The following image shows the creation of an index using a remote storage option, such as Amazon S3.
+
+![Searchable Snapshots](/assets/media/blog-images/search_snap.png){: .img-fluid}
+
+## Enabling searchable snapshots in OpenSearch
+
+To enable searchable snapshots, reference the [instructions](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/) in the OpenSearch documentation.
+
+## Using a snapshot for search
+
+OpenSearch cluster nodes retrieve data from a snapshot and then cache it to the node's local storage. You can configure the amount of storage available for caching snapshot data or use the system defaults. Any subsequent searches on the cached data perform better because they are served from the node's local storage, or cache, without the need to redownload the segment data from the snapshot.
+
+For a search request on data in an uncached snapshot, the data is downloaded from the snapshot's object store. If a node's cache is full, OpenSearch clears data segments from the cache using the Least Recently Used (LRU) caching strategy, removing the least recently used segments first.
+
+## Performance
+
+When cached data is searched repeatedly, searchable snapshots provide consistent search performance and throughput. Performance is slow the first time the snapshot data is accessed, but subsequent searches are comparable in performance to regular OpenSearch cluster searches. Note, however, that the ratio of cache size to the combined size of the searchable snapshot, along with the search pattern, will factor into performance. The more data the cache misses, the lower the overall performance. In the next section, we cover the performance results of our benchmark testing.
```

Comment:
I would re-word the whole paragraph with a summary of Searchable Snapshot performance. Some suggested content:
Comparing the performance of Searchable Snapshots queries and local index queries on the StackOverflow dataset, the initial run of Searchable Snapshots took approximately twice as long as local index. However, subsequent queries showed similar performance for both Searchable Snapshots and local index, as Searchable Snapshots leveraged the cached data.